### ISSUE TITLE
How to use  eval_image_classifier.py  to predict  my images?

### ISSUE THREAD

**Issue by phybrain** [pudae/tensorflow-densenet#8]
I'm not very familiar with this code enough，can you help me？

**Comment by pudae** [comment]
Here is sample code. [link](https://gist.github.com/pudae/3b37e77d6258cff0fe5fcfa3a71a1fb5)
Usage: python predict_image.py --image=./sample_image.jpg --checkpoint_path=./tf-densenet121.ckpt

**Comment by mit10000** [comment]
Pudae, 
Thanks for sharing.
One further question.
In your code, the output logits is defined by

logits, _ = network_fn(images)

In deployment, if we don't want to disclose network_fn source codes, instead we use

logits=tf.get_default_graph().get_tensor_by_name('densenet169/logits/BiasAdd:0')

We can do prediction, but the results are different. Could you please tell what cause this problem?
